code/game/objects/items/stacks/sheets/sheet_types.dm

```diff
@@ -13,6 +13,7 @@
  */
 var/global/list/datum/stack_recipe/metal_recipes = list ( \
 	new/datum/stack_recipe("stool", /obj/structure/chair/stool, one_per_turf = 1, on_floor = 1), \
+	new/datum/stack_recipe("bar stool", /obj/structure/chair/stool/bar, one_per_turf = 1, on_floor = 1), \
```

Comment:
![aaa](https://cloud.githubusercontent.com/assets/6282424/15952006/a70114e4-2e93-11e6-9199-798eda6ed7d8.gif)
if this is not testing, well, I don't know what is.